Packs/CommonPlaybooks/Playbooks/playbook-Block_Indicators_-_Generic_v3.yml

```diff
@@ -0,0 +1,780 @@
+      playbookName: Block IP - Generic v3
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "#none#":
+      - "2"
+    scriptarguments:
+      AkamaiNetworkListID:
+        simple: ${inputs.AkamaiNetworkListID}
+      AutoCommit:
+        simple: $}inputs.AutoCommit{
+      CiscoFWSource:
+        simple: ${inputs.CiscoFWSource}
+      CustomBlockRule:
+        simple: ${inputs.CustomBlockRule}
+      DAG:
+        simple: ${inputs.DAG}
+      IP:
+        simple: ${inputs.IP}
+      InputEnrichment:
+        simple: ${inputs.InputEnrichment}
+      InternalRange:
+        simple: ${inputs.InternalRange}
+      LogForwarding:
+        simple: ${inputs.LogForwarding}
+      RuleDirection:
+        simple: ${inputs.RuleDirection}
+      RuleName:
+        simple: ${inputs.RuleName}
+      SiteName:
+        simple: ${inputs.SiteName}
+      StaticAddressGroup:
+        simple: ${inputs.StaticAddressGroup}
+      Tag:
+        simple: ${inputs.Tag}
+      UserVerification:
+        simple: ${inputs.UserVerification}
+    separatecontext: true
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 0
+    view: |-
+      {
+        "position": {
+          "x": 2200,
+          "y": 340
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "14":
+    id: "14"
+    taskid: e80448b5-aa76-49e1-8d42-8d64f864f340
+    type: title
+    task:
+      id: e80448b5-aa76-49e1-8d42-8d64f864f340
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 1125,
+          "y": 835
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "15":
+    id: "15"
+    taskid: fe262809-0bda-4f4d-873a-a1b9d441f5cb
+    type: regular
+    task:
+      id: fe262809-0bda-4f4d-873a-a1b9d441f5cb
+      version: -1
+      name: Tag bad indicators
+      description: commands.local.cmd.set.indicators
+      script: Builtin|||setIndicators
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      "#none#":
+      - "14"
+    scriptarguments:
+      indicatorsValues:
+        simple: ${inputs.IP},${inputs.URL},${inputs.Username},${inputs.MD5} ,${inputs.SHA256}
+      tags:
+        simple: Blocked Indicator In Systems
```

Comment:
That doesn't sound right, was it approved by the tech-writers or somebody in product?